robot/impl/local_robot_test.go

```diff
@@ -3032,9 +2928,6 @@ func TestDependentAndOrphanedResources(t *testing.T) {
 
 func TestModuleDebugReconfigure(t *testing.T) {
 	ctx := context.Background()
-	// We must use an Info level observed test logger to avoid testmodule
-	// inheriting debug mode from the module manager.
-	logger, logs := rtestutils.NewInfoObservedTestLogger(t)
```

Comment:
I set the logger level in the test helper to `INFO` to accommodate this test. I think this is the correct approach generally since we shouldn't need to test `DEBUG` level logs IMO.